spec/controllers/verify/confirmations_controller_spec.rb

```diff
@@ -34,7 +34,7 @@
       let(:saml_authn_request) { sp1_authnrequest }
 
       before do
-        subject.session[:saml_request_url] = saml_authn_request
+        subject.session[:user_return_to] = saml_authn_request
```

Comment:
yes, I am still trying to understand why `stored_location_for(user) ` works, then, since under the hood that is looking at `session[:user_return_to]`..